Widgets/Widgets/PictureOfTheDayWidget.swift

```diff
@@ -170,126 +170,134 @@ struct PictureOfTheDayEntry: TimelineEntry {
 
 struct PictureOfTheDayProvider: TimelineProvider {
 
-	// MARK: Nested Types
-	
-	public typealias Entry = PictureOfTheDayEntry
+    // MARK: Nested Types
+
+    public typealias Entry = PictureOfTheDayEntry
+
+    // MARK: Properties
 
-	// MARK: Properties
-	
-	private let dataStore = PictureOfTheDayData.shared
+    private let dataStore = PictureOfTheDayData.shared
 
-	// MARK: TimelineProvider
+    // MARK: TimelineProvider
 
-	func placeholder(in: Context) -> PictureOfTheDayEntry {
-		return dataStore.placeholderEntry
-	}
+    func placeholder(in: Context) -> PictureOfTheDayEntry {
+        return dataStore.placeholderEntry
+    }
     
-	func getTimeline(in context: Context, completion: @escaping (Timeline<PictureOfTheDayEntry>) -> Void) {
-		dataStore.fetchLatestAvailablePictureEntry { entry in
-			let currentDate = Date()
-			let timeline = Timeline(entries: [entry], policy: .after(currentDate.dateAtMidnight() ?? currentDate))
-			completion(timeline)
-		}
-	}
-
-	func getSnapshot(in context: Context, completion: @escaping (PictureOfTheDayEntry) -> Void) {
-		dataStore.fetchLatestAvailablePictureEntry(usingCache: context.isPreview) { entry in
-			completion(entry)
-		}
-	}
+    func getTimeline(in context: Context, completion: @escaping (Timeline<PictureOfTheDayEntry>) -> Void) {
+        dataStore.fetchLatestAvailablePictureEntry { entry in
+            let currentDate = Date()
+            let nextUpdate: Date
+            let isError = (entry.imageDescription == dataStore.sampleEntry.imageDescription)
```

Comment:
Because the sample entry's `imageDescription` is localized (which I think might be the wrong approach for this particular widget), I think we need to consider the error state using something else. Maybe `imageURL` instead? It's kind of strange because it's like, what do we consider an error state for this particular widget? Using `image` directly might be the most "correct" approach to cover multiple "error" states.

Reply:
I'll use whatever you recommend for error states for these - Though I think we need a check on an optional or a check on a non-class (as the objects won't be identical for the images). Image is a struct, so I think this would compare just fine, right?

Reply:
Sorry if I was unclear - I just meant the `UIImage` `image` directly, comparing if their data is equivalent. Conceptually here, I think error is either the failure to have an image for the entry or the entry image being the same as the sample entry's image.

```swift
entry.image == nil || entry.image == dataStore.sampleEntry.image
```

Reply:
Updated the error check. Thanks!